### ISSUE TITLE
maybe a bug of freeze_at 2 in backbone?

### ISSUE THREAD

**Issue by Wei-i** [hzhupku/DCNet#15]
https://github.com/hzhupku/DCNet/blob/58b8b09bf6b242ff09239c6489fb814c33d9a19b/maskrcnn_benchmark/config/defaults.py#L105

**Comment by Wei-i** [comment]
Hi, I am confused about this default config because a support image consists of (image, mask) which is 4-d, so we can not use pretrained ResNet weights and add a `conv2` in the `BaseStem`.
However,  `FREEZE_CONV_BODY_AT=2` , contains `BaseStem`. 
After I debug this line, when we use this freeze config, self.conv2(x).requires_grad is false. This means we can not train this weight.

What is wrong with my view? Hope you can help me.

**Comment by wsjxdy** [comment]
I followed the install.md build environment to build four or five times, and every time I encountered ImportError:
_/DCNet-main/maskrcnn_benchmark/_C.cpython-36m-x86_64-linux-gpu.so: undefined symbol: __cudaPopCallConfiguration. _ But I am completely I installed it according to install.md. The cuda version of my computer is 9.0.176, and gcc tried 5.5 and 7.5, but they all reported the same error. What are the specific torch, cuda, torchvision, gcc and other environments you are using?

**Comment by wsjxdy** [comment]
@Wei-i Looking forward to your quick reply!

**Comment by Wei-i** [comment]
It seems that cuda version  may be not compatible with your environment.

**Comment by Wei-i** [comment]
Only difference of my config is CUDA 10.1.
The others follow the official install.md.

**Comment by wsjxdy** [comment]
Thank you very much for your kindness!I will try again.

**Comment by wsjxdy** [comment]
I tried to switch the cuda soft connection in /usr/local to cuda10.1, the program "from maskrcnn_benchmark import _C" still reports the original error, that is, undefined symbol: __cudaPopCallConfiguration. I use the command "python setup.py build develop" No error will be reported.
If I use cuda10.1 from the build environment,, when using "python setup.py install --cuda_ext --cpp_ext" to compile apex, the error "cuda version does not match, only supports cuda9.0.176" will be reported. 
Could you please confirm that your torch and torchvision 1.1.0 and 0.3 respectively? Thank you for your help! @Wei-i

**Comment by Wei-i** [comment]
"cuda version does not match, only supports cuda9.0.176"  seems that your cuda or cudatoolkit may be not compatible instead of the version of torch.
I use torch 1.4.0 with torchvision 0.5.
I think CUDA 9.0 may be  also compatible, for more details, please refer to official install.md or open the other issue in maskrcnn_benchmark repo.
